features/help.feature

```diff
@@ -62,15 +70,186 @@ Feature: Get help about WP-CLI commands
     Then the return code should be 1
     And STDERR should be:
       """
-      Error: 'non-existent-command' is not a registered wp command.
+      Error: 'non-existent-command' is not a registered wp command. See 'wp help'.
+      """
+    And STDOUT should be empty
+
+    When I try `wp help non-existent-command --path=/nowhere`
+    Then the return code should be 1
+    And STDERR should be:
+      """
+      Warning: This does not seem to be a WordPress install.
+      If the command 'non-existent-command' is in a plugin or theme, pass --path=`path/to/wordpress`.
+      Error: 'non-existent-command' is not a registered wp command. See 'wp help'.
```

Comment:
If we definitely know this isn't a WordPress install, we don't need to include `If the command 'non-existent-command' is in a plugin or theme, pass --path=path/to/wordpress`